platform/core/src/main/java/org/corfudb/platform/core/Value.java

```diff
@@ -0,0 +1,108 @@
+package org.corfudb.platform.core;
```

Comment:
the package name platform and core does not sound right. 
i think we should start  like org.corfudb.runtime.keyvaluestore. or something to that effect. 
Please work with @Maithem @zalokhan @annym to define the right naming convention

Reply:
I wouldn't like to use `runtime` in a package name. Platform doesn't know anything about runtime. This name seems too narrow/specific to me.
So, let's discuss tomorrow

Reply:
As per our discussion please make the necessary changes.